src/flask_session/sessions.py

```diff
@@ -60,7 +60,7 @@ class SqlAlchemySession(ServerSideSession):
 class SessionInterface(FlaskSessionInterface):
 
     def _generate_sid(self):
-        return str(uuid4())
+        return secrets.token_urlsafe(128//8)
```

Comment:
Maybe the amount of entropy in the session should be configurable. I wonder how 128 compares to other web frameworks?

Django sessions are 32 mixed case alphanumeric characters giving 165 bits of entropy, for instance.

Reply:
Interesting. I also noted that the secrets module documentation recommends 256 bits for tokens in general [https://docs.python.org/3/library/secrets.html](https://docs.python.org/3/library/secrets.html). 

I'm also guessing with flask-session being server-side, all of the payload is server-side so we don't need to worry too much about bandwidth. Although I'm sure saving a little is nice.

I also noticed Django limit their characters to alphanumeric to avoid any injection possibility in event of secret_key being compromised and using the pickle module. They have switched to JSON serialization by default.[https://github.com/django/django/blob/f7389c4b07ceeb036436e065898e411b247bca78/docs/releases/1.5.3.txt#L33](https://github.com/django/django/blob/f7389c4b07ceeb036436e065898e411b247bca78/docs/releases/1.5.3.txt#L33)
[https://docs.djangoproject.com/en/4.2/topics/http/sessions/](https://docs.djangoproject.com/en/4.2/topics/http/sessions/)
[https://docs.python.org/3/library/pickle.html#module-pickle](https://docs.python.org/3/library/pickle.html#module-pickle)